packages/mux-player/src/index.ts

```diff
@@ -172,6 +173,27 @@ class MuxPlayerInternal {
     }
   }
 
+  _setUpLiveFirstPlay(el: MuxPlayerElement) {
+    if (el.video?.hls) {
+      const Hls: any = el.video.hls.constructor;
+      if (!el.autoplay) {
+        el.video.hls.on(Hls.Events.LEVEL_LOADED, (e, data) => {
+          const isLive = data.details.live;
+
+          if (isLive) {
+            el.video.addEventListener(
+              "play",
+              () => {
+                el.video.currentTime = el.video.hls.liveSyncPosition;
+              },
+              { once: true }
+            );
+          }
+        });
+      }
+    }
+  }
```

Comment:
could this already be moved to mux-video? 

this is needed because hls.js starts loading, filling the buffer right away on player load, I wonder if that should happen with live streams if autoplay is not enabled?

Reply:
Yeah, definitely could be moved to `mux-video`. This is a draft PR, plus, if we wanted to augment the `autoplay` attribute, we'll want to figure out whether we want to do it in mux-player or mux-video, or both, since mux-video currently only accepts a boolean autoplay attribute.